modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -188,6 +190,26 @@ def buildProject(Map options = [:]) {
       }
     }
 
+    if (options.publishingE2eTests == true) {
+      setEnvar("FULL_COMMIT_HASH", sh(
+        script: "git rev-parse HEAD",
+        returnStdout: true
+      ))
+      stage("End-to-end tests") {
+        if ( env.PUBLISHING_E2E_TESTS_APP_PARAM == null ) {
```

Comment:
@kevindew I've decided to use an if/else statement to set `appCommitishName` and `testBranch` as I found using a ternary statement exceeded the line length too much.